src/platform/intel/cavs/platform.c

```diff
@@ -367,8 +371,23 @@ int platform_init(struct sof *sof)
 	trace_point(TRACE_BOOT_PLATFORM_IDC);
 	idc_init();
 
+	/* initialize the SPI slave */
+	spi = sspi_get(SOF_SPI_INTEL_SLAVE);
+	if (spi == NULL)
+		return -ENODEV;
+
+	ret = sspi_probe(spi);
+	if (ret < 0)
+		return ret;
+
+	/* initialize the SPI-SLave module */
+	ret = spi_slave_init(spi, SOF_SPI_INTEL_SLAVE);
+	if (ret < 0)
+		return ret;
+
 	/* Initialize DMA for Trace*/
+#if !defined(CONFIG_SUECREEK)
```

Comment:
it's more that we need a define to disable the trace, and that's set by default in SueCreek.
I'd love to disable the stupid trace, it's been problematic for a year now...